### DISCUSSION TITLE
Anti-Corruption Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#39] [Added]
### Type name

~Foreign Corrupt Practices Act Policy~
Anti-Corruption Policy https://github.com/OpenTermsArchive/terms-types/discussions/39#discussioncomment-10187740

### Alternative type names

~Foreign Corrupt Practices Act (FCPA) Compliance Policy~
~UK Bribery Act 2010 (UKBA) Compliance Policy~
Anti-bribery Policy https://github.com/OpenTermsArchive/terms-types/discussions/39#discussioncomment-10187911

### Writer

service provider

### Audience

~employees~
any person working directly or indirectly for the service provider https://github.com/OpenTermsArchive/terms-types/discussions/39#discussioncomment-10187740

### Object

~legal and compliance obligations set forth FCPA~
risk mitigation of involvement in bribery https://github.com/OpenTermsArchive/terms-types/discussions/39#discussioncomment-10187740

### Examples

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | TransUnion | [URL 🔗](https://www.transunion.com/legal/fcpa-policy) | Foreign Corrupt Practices Act (FCPA) Policy | 🇺🇸 |
| 2 | ExxonMobil | [URL 🔗](https://corporate.exxonmobil.com/-/media/global/files/policy/anti-corruption-legal-compliance-guide.pdf)  | Anti-Corruption Legal Compliance Guide  | 🇺🇸 |
| 3 | Kaspersky | [URL 🔗](https://www.kaspersky.com/anti-corruption-policy) | GLOBAL ANTI-CORRUPTION COMPLIANCE POLICY  | 🌐 |
| 4 | Airbus | [URL 🔗](https://www.airbus.com/sites/g/files/jlcbta136/files/2021-07/Airbus-Ethics-and-Compliance-Anti-Corruption-Policy.pdf) | Anti-CorruptionPolicy | 🌐 |
| 5 | Carlsberg | [URL 🔗](https://www.carlsberggroup.com/media/lavfbez3/anti-bribery-corruption-policy_eng_aug-2019.pdf) | ANTI-BRIBERY & CORRUPTION POLICY | 🌐 |
| 6 | Amundi | [URL 🔗](https://legroupe.amundi.com/files/nuxeo/dl/725b1ee7-6b67-4a35-9f2b-ce500664f439) | POLITIQUE ANTI-CORRUPTION | 🇫🇷 |
| 7 | Whirlpool | [URL 🔗](https://1c938699858ebac8919f-943ad49fa0bf56534d6539e94192646e.ssl.cf5.rackcdn.com/files/Gifts%20and%20Entertainment%20Guidelines%20(12-16-19).pdf) | GIFTS & ENTERTAINMENT GUIDELINES | 🌐 |

### References

- [USA Foreign Corrupt Practices Act](https://www.justice.gov/criminal/criminal-fraud/foreign-corrupt-practices-act)
- [UK Bribery Act 2010](https://www.legislation.gov.uk/ukpga/2010/23/contents)
- [ISO 37001 Anti-bribery management systems](https://www.iso.org/iso-37001-anti-bribery-management.html)
- [Loi n°2016-1691 du 9 décembre 2016 relative à la transparence, à la lutte contre la corruption et à la modernisation de la vie économique, dite « Sapin II »](https://www.legifrance.gouv.fr/loda/id/JORFTEXT000033558528)

**Comment by MattiSG** [comment]
These anti-corruption policies are interesting and relevant. The “foreign corrupt practices act” legal reference is only valid in the USA, and should be broadened.

Example 1 for example specifically includes:

> While this policy addresses the FCPA specifically, TransUnion and TransUnion employees must comply with all laws, including all anti-bribery and anti-corruption laws, of countries or nations in which TransUnion operates.

Example 2 states:

>  The FCPA was the only transnational anti-corruption statute in the world for many years, but that is no longer the case. Since 1996, more than 150 countries have signed one or more of a series of multilateral conventions under the auspices of organizations such as the Organization for Economic Cooperation and Development (OECD), the Organization of American States (OAS), the Council of Europe (COE), the United Nations, and the African Union.

Example 3 states:

> This Policy focuses on anti-corruption laws, including the U.K. Bribery Act 2010 as amended (the “UKBA”), the anti-corruption laws of the Russian Federation, the U.S. Foreign Corrupt Practices Act of 1977 as amended (the “FCPA”), and other laws to which the Company (as defined in clause 1.4 below) is subject.

Thus:

## Type name

- Anti-Corruption Policy

## Alternative types names

- Foreign Corrupt Practices Act (FCPA) Compliance Policy
- UK Bribery Act 2010 (UKBA) Compliance Policy

## Object

risk mitigation of involvement in bribery

## Audience

As with https://github.com/OpenTermsArchive/terms-types/discussions/40#discussioncomment-10179422: I would prefer some more descriptive phrasing, such as “any person working directly or indirectly for the service provider”.

## References

I'm in favour of keeping only the official reference from DoJ and not the equivalent description in the Wikipedia article, so the references could be:

- [USA Foreign Corrupt Practices Act](https://www.justice.gov/criminal/criminal-fraud/foreign-corrupt-practices-act)
- [UK Bribery Act 2010](https://www.legislation.gov.uk/ukpga/2010/23/contents)

**Reply by clementbiron** [reply]
thank you, I've taken your feedback into account!

**Reply by MattiSG** [reply]
The alternate type names I suggested are not relevant, we should mention the legal references only in the “references” section. The alternative type names should be for synonyms, such as, here, “Anti-Bribery Policy”.

**Comment by clementbiron** [comment]
Additional reference:
[ISO 37001 Anti-bribery management systems](https://www.iso.org/iso-37001-anti-bribery-management.html)

**Comment by MattiSG** [comment]
Additional reference:

[Loi n°2016-1691 du 9 décembre 2016 relative à la transparence, à la lutte contre la corruption et à la modernisation de la vie économique, dite « Sapin II »](https://www.legifrance.gouv.fr/loda/id/JORFTEXT000033558528)

**Comment by MattiSG** [comment]
Additional example:

Amundi https://legroupe.amundi.com/files/nuxeo/dl/725b1ee7-6b67-4a35-9f2b-ce500664f439 🇫🇷

**Comment by MattiSG** [comment]
Another example:

Whirlpool | https://1c938699858ebac8919f-943ad49fa0bf56534d6539e94192646e.ssl.cf5.rackcdn.com/files/Gifts%20and%20Entertainment%20Guidelines%20(12-16-19).pdf | GIFTS & ENTERTAINMENT GUIDELINES | 🌐

**Comment by clementbiron** [comment]
This terms type suggestion has been accepted and will soon be added.
